front_end/Inbox/Message.js

```diff
@@ -1,52 +1,54 @@
 import React, {Component} from 'react';
-import {
-    Text,
-    TouchableHighlight,
-    View
-} from 'react-native';
+import {Text, TouchableHighlight, View} from 'react-native';
 
 import Icon from '../Utility/Icon';
-import styles from '../Static/style';
+import styles from '../Static/main_style';
 import inbox_style from '../Static/inbox_style';
 
 export default class Message extends Component {
```

Comment:
Is this a class for a single message, or all messages between two people? If the latter, perhaps a more suggestive name would be helpful.

Reply:
May you please suggest a better name? I was asked to change it to main_style from Eddy. Perhaps matching_style?

Reply:
I'm talking about the class. 

export default class Message ... {

Reply:
Changed Message to SingleMEssage on InboxOne.js and Message.js